main/src/main/scala/org/clulab/processors/clu/PostProcessor.scala

```diff
@@ -2,26 +2,27 @@ package org.clulab.processors.clu
 
 import org.clulab.processors.Sentence
 
+import java.util.regex.Pattern
+
 object PostProcessor {
   //
   // Patterns for post-processing corrections
   //
-  val VERSUS_PATTERN = """(?i)^vs\.?$""".r
+  val VERSUS_PATTERN = Pattern.compile("""(?i)^vs\.?$""")
```

Comment:
Just for my own curiosity, why is this better than `.r`?

Reply:
It may not be for a Scala regex.  I see that it keeps track internally of its own compiled pattern, which I had forgotten.  On the other hand, findFirstIn(string).nonEmpty is pretty ugly compared to a boolean matches().  If they added that, and they have in Scala 2.13+, I would probably switch back.  Then it would be VERSUS_REGEX instead.  At least now the casual reader can know that the pattern is compiled and the small overhead of the Scala regex is skipped.  Perhaps it's time to re-evaluate my standard procedure.

Reply:
Ok!